pkgs/development/tools/literate-programming/nuweb/default.nix

```diff
@@ -0,0 +1,37 @@
+{stdenv, fetchurl, tex}:
+
+stdenv.mkDerivation rec{
+
+  name = "nuweb-${version}";
+  version = "1.58";
+
+  src = fetchurl {
+    urls = [ "http://downloads.sourceforge.net/project/nuweb/${name}.tar.gz" ];
```

Comment:
No special reason indeed. I was playing with the noweb expression, using it as a model.